docs/adr/adr-010-incentivized-testnet-monitoring.md

```diff
@@ -167,14 +168,44 @@ The Prometheus docs state the following with regard to [Denial of Service](https
 
 So if we are concerned about the public leaderboard crashing the Prometheus instance that we use for internal dashboards, we may want to host two separate instances. This seems feasible by configuring OTEL Collector to export to two different Prometheus instances. This is a one way door, I suggest sticking with one instance because Grafana Cloud guarantees 99.5% uptime.
 
-### Why doesn't the Celestia team host OTEL Collectors for node operators?
+### Which actor should run OTEL Collector(s) during the incentivized testnet?
 
-1. Node operators will lose the ability to monitor their own celestia-node. Since opentelemetry-go only supports configuring one exporter ( [open-telemetry/opentelemetry-go#3055](https://github.com/open-telemetry/opentelemetry-go/issues/3055)), if node operators were obligated to export metrics to a Celestia team managed OTEL Collector endpoint, they wouldn't be able to export to their own OTEL Collector (and by proxy any telemetry platforms they wish to use). This violates the desideratum:
+#### Scenario A: Node operators
+
+Pros
+
+- This deployment architecture is more representative of mainnet where node operators will run their own telemetry stack to monitor their node. Exposing node operators to OTEL Collector during incentivized testnet allows them to practice this deployment architecture prior to mainnet.
+- Node operators will have an "incentive" to maintain high uptime for their OTEL Collector.
+
+Cons
+
+- Additional operational burden for incentivized testnet participants. We can mitigate this concern by providing easy install steps and scripts.
+
+#### Scenario B: Celestia team
+
+Pros
+
+- It will be easier for nodes to participate if they only have to deploy one piece of software (celestia-node) and not two (celestia-node and OTEL Collector).
+
+Cons
+
+- Node operators will lose the ability to monitor their own celestia-node. Since opentelemetry-go supports configuring only one exporter ( [open-telemetry/opentelemetry-go#3055](https://github.com/open-telemetry/opentelemetry-go/issues/3055)), if node operators were obligated to export metrics to a Celestia team managed OTEL Collector endpoint, they wouldn't be able to export to their own OTEL Collector (and by proxy any telemetry platforms they wish to use). This violates the desideratum:
```

Comment:
They don't lose, as we discussed in Slack. They can run a node pointing to the local Otel Collector configured to send metrics to Celestia infra and their own monitoring.

Btw, is it possible for Collector to send metrics to another Collector? Because of OTLP it should be, but still, there might be some caveats. I am talking about the above case where a local Collector sends metrics to the global network Collector. If not, the local Collector can send data directly to Prometheus, etc

Reply:
> Btw, is it possible for Collector to send metrics to another Collector?

Yes. This is often the recommended approach. See https://lightstep.com/opentelemetry/collecting-and-exporting-data

Agreed they can run their own OTEL collector to perform their own monitoring. Scenario B was meant to describe celestia-node -> OTEL Collector managed by Celestia team and Scenario C was meant to describe optionality of node-operator deployed OTEL Collector. I'll add to this section that node operators don't lose ability to monitor their node if they deploy their own OTEL Collector.

Reply:
Done in https://github.com/celestiaorg/celestia-node/pull/922/commits/7ffa115337aa21e62177483e68c64763041cf615